FirebaseCore/Sources/Private/FIRAppInternal.h

```diff
@@ -20,9 +20,9 @@
 @protocol FIRLibrary;
 
 /**
- * The internal interface to FIRApp. This is meant for first-party integrators, who need to receive
- * FIRApp notifications, log info about the success or failure of their configuration, and access
- * other internal functionality of FIRApp.
+ * The internal interface to `FirebaseApp`. This is meant for first-party integrators, who need to
```

Comment:
The FirebaseCore Private header are currently only used by internal Objective C clients, but will probably soon be internally public for consumption by Firebase Swift implemented SDKs - so thanks! cc: @ryanwilson